javascript/java.js

```diff
@@ -0,0 +1,10 @@
+function dogInfo(name, breed, cost) {
+  alert('More on this dog:\n' + name + '\n' + breed + '\n' + cost + ' to adopt');
+}
+
+let total = 0;
+
+function totalCost(x) {
+  let newTotal = total += x;
+  alert('$' + newTotal);
+}
```

Comment:
Functionally this works, but in practice there is some wasted code here. For lines 8 and 9 I would change it to the following:
 ` total += x;`
  `alert('$' + total);`
The reason is the variable newTotal is sort of redundant, we already have a variable 'total' that we are using to keep track of the changes in price. newTotal only exists for an instant while the function runs (because it is block scoped or 'defined' inside the function) and is then removed from memory. 
Hopefully this will make more sense as we continue more Javascript exercises!